daemon/oci_windows.go

```diff
@@ -25,11 +25,31 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	// In base spec
 	s.Hostname = c.FullHostname()
 
+	if err := daemon.setupSecretDir(c); err != nil {
+		return nil, err
+	}
+
 	// In s.Mounts
 	mounts, err := daemon.setupMounts(c)
 	if err != nil {
 		return nil, err
 	}
+
+	if !c.HasBeenStartedBefore && len(c.SecretReferences) > 0 {
+		if err := daemon.Mount(c); err != nil {
```

Comment:
Is there a by-design guarantee that the container file system is mounted here in the non-Hyper-V case? It seems strange to me to start modifying the container file system without first ensuring it is mounted. Is that guarantee documented anywhere, or just implicit in the design?